### ISSUE TITLE
Test failure: 'root' is deprecated, use 'global'

### ISSUE THREAD

**Issue by saschanaz** [w3c/respec#1458]
https://travis-ci.org/w3c/respec/jobs/335038255

Not sure what happened there...

**Comment by saschanaz** [comment]
`ReSpec error: (node:6910) DeprecationWarning: 'root' is deprecated, use 'global'`

This seems the cause. Strangely, I rollbacked to the last known good commit and it still happens there.

**Comment by saschanaz** [comment]
I think I got the cause. cc: @nileshgulia1

handlebars.js from dependencies starts with:

```js
(function webpackUniversalModuleDefinition(root, factory) {
	if(typeof exports === 'object' && typeof module === 'object')
		module.exports = factory();
	else if(typeof define === 'function' && define.amd)
		define([], factory);
	else if(typeof exports === 'object')
		exports["Handlebars"] = factory();
	else
		root["Handlebars"] = factory();
})(this, function() {
```

Here, the UMD exporting code uses a variable `root`. Our build system somehow convert it to:

```js
root = this,
factory = function() { /* ... */ },
"object" == typeof exports && "object" == typeof module ? module.exports = factory() : "function" == typeof define && define.amd ? define("handlebars.runtime", [], factory) : "object" == typeof exports ? exports.Handlebars = factory() : root.Handlebars = factory(),
```

This made Node.js angry where `root` is a deprecated alias of `global`.

**Comment by saschanaz** [comment]
Still I have no idea why this is suddenly a problem, and only a problem for the last test.

**Comment by nileshgulia1** [comment]
Nice work @saschanaz  I'll try to figure out the rest.

**Comment by saschanaz** [comment]
Generated build result is different with the main repo, probably affected by some change in the dependencies.

**Comment by marcoscaceres** [comment]
Just fyi, I’m in the middle of moving house, so will be a bit slow to respond to things till next week! Thanks nonetheless for looking into it.

**Comment by saschanaz** [comment]
Oops, I was just to request uploading your local `package-lock.json` to see what update causes this issue :smile:

It seems `yarn.lock` is now too old.

**Comment by marcoscaceres** [comment]
Yeah, sorry there is both... npm 5 was... not a great release... so I had to switch between yarn and npm a few times.

**Comment by saschanaz** [comment]
No worry, I got a fix 😁 #1461
